src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -151,6 +152,54 @@ const WorkspaceAccessError = () => {
   ])
 }
 
+const checkBucketAccess = withErrorReporting('Error checking bucket access', async (signal, namespace, name) => {
+  try {
+    await Ajax(signal).Workspaces.workspace(namespace, name).checkBucketReadAccess()
+    return true
+  } catch (error) {
+    if (error.status === 403) {
+      notify('error', div([
+        'The Google Bucket associated with this workspace is currently unavailable. This should be resolved shortly. If this persists for more than an hour, please write our',
+        link({
+          target: '_blank',
+          href: 'https://broadinstitute.zendesk.com/hc/en-us',
```

Comment:
What if this link opened the "Contact Us" support modal? That way users stay on Terra and there's less of a disruption to their work in order to get help. The contact us is an atom so it can be triggered from anywhere.